src/util/scrollTo.js

```diff
@@ -30,21 +30,25 @@ export default function scrollTo(scrollTo, { element }, tooltipLayer) {
     // the center of the target element or tooltip.
 
     if (top < 0 || element.clientHeight > winHeight) {
-      window.scrollBy(
-        0,
-        rect.top -
+      window.scrollBy({
```

Comment:
Would be great if we add some Cypress tests

Reply:
Agreed. What would you test in this case? It still uses the same function window.scrollBy()

Reply:
Sorry for the delay @sammuell. We usually have a delay() between each step in the Cypress tests to make sure the transition is complete. Now that the transition can be enabled/disabled, we can probably have a test _without_ that delay(), take a screenshot and that should capture the tooltip in its correct position. Would that be a useful test case?